extensions/hacker-news-top-stories/package.json

```
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "hacker-news-top-stories",
+  "title": "Hacker News Top Stories",
+  "description": "A menubar extension to view top stories on Hacker News based on a configurable points threshold.",
+  "icon": "icon.png",
+  "author": "KevinBatdorf",
+  "contributors": [
+    "pernielsentikaer"
+  ],
+  "categories": [
+    "News"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "view-top-stories",
+      "title": "Hacker News Top Stories Menu Bar",
+      "description": "View Hacker News stories with 500+ points submitted over the last 24 hours.",
```

Maybe something like this?

```suggestion
      "description": "View Hacker News stories based on a configurable points threshold which are submitted over the last 24 hours.",
```